pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -17,8 +17,12 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
+const (
+	offchainReporting = "offchain_reporting"
```

Comment:
Maybe call it "ocrCapabilityType"?

Reply:
Hmmm... it doesn't really feel like a type though. WDYT of `ocrCapabilityId`?

Reply:
You're right, that's not type but ID, thanks.